src/routes/news/components/NewsListItem.js

```diff
@@ -110,44 +112,54 @@ export const ReadMore: StyledComponent<
   margin-top: 5px;
   color: ${props =>
     props.isTunews
-      ? '#0279a6'
+      ? props.theme.colors.tunewsThemeColor
       : props.theme.colors.themeColor};
 `
 
 class NewsListItem extends React.PureComponent<PropsType> {
   render () {
     const {
-      newsItem: { title, content, message },
+      newsItem,
+      newsItem: { title },
       language,
       navigateToNews,
       theme,
       t,
       isTunews
     } = this.props
-    const newsContent = content || message
+
+    let content = ''
+    if (newsItem.content) { content = newsItem.content }
+    if (newsItem.message) { content = newsItem.message }
```

Comment:
```suggestion
    const content =newsItem.content ? newsItem.content : newsItem.message
```

Reply:
Unfortunately, tried this before but flow type fails when i do this fix

Reply:
Allright, no problem